dart/collision/fcl/FCLTypes.hpp

```diff
@@ -55,25 +79,97 @@ template <class T> using fcl_shared_ptr = boost::shared_ptr<T>;
 template <class T> using fcl_weak_ptr = boost::weak_ptr<T>;
 #endif
 
-namespace dart {
-namespace collision {
+namespace dart
+{
+namespace fcl
+{
+#if FCL_VERSION_AT_LEAST(0,6,0)
+// Geometric fundamentals
+typedef ::fcl::Vector3<double> Vector3;
```

Comment:
The current convention with FCL specific types for is using a prefix. Could you remove `fcl` namespace and prepend `FCL` for the type aliasing (e.g., `using FCLVector3 = ::fcl::Vector3<double>` or `using FCLVector3 = ::fcl::Vector3d`)?